agent/src/beerocks/slave/agent_db.h

```diff
@@ -0,0 +1,76 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "task_pool.h"
+
+#include <iostream>
+#include <memory>
+#include <mutex>
+
+using namespace beerocks_message;
+using namespace son;
+using namespace beerocks;
```

Comment:
```suggestion
namespace beerocks {
```
*Everything* should be in the beerocks namespace.

Reply:
ok. I removed the others for now. I think I added it because of the task infra which I removed, so...